vue/sbc-common-components/src/components/SbcHeader.vue

```diff
@@ -20,6 +20,31 @@
           <!-- Product Selector -->
           <sbc-product-selector v-if="showProductSelector" />
 
+          <!-- What's New -->
+          <v-btn
+            text
+            dark
+            large
+            width="150"
+            aria-label="whatsnew"
+            attach="#appHeader"
+            @click.stop="notificationPanel=true"
+            v-if="!isAuthenticated && notificationCount > 0 && isWhatsNewOpen"
+          >
+            <v-badge
+              dot
+              overlap
+              offset-y="-5"
+              offset-x="10"
+              color="notificationUnreadPriorityCount > 0 ? error : blue"
```

Comment:
Does this work?
1. I'm surprised you don't need to use `:color=`
2. I'm surprised you don't need to use `'error' : 'blue'`

Reply:
This is working. :) But I will change it as your recommended.